sass/base/_base.scss

```diff
@@ -35,13 +35,13 @@ body {
   }
 }
 
-html[lang="ar"] body {
+html[lang="ar"] body,
+html[lang="ru"] body {
   font-family: $cd-font--noto;
```

Comment:
Noto Sans doesn't handle Arabic, so that would result in the system fonts being used for Arabic.